Makefile.include

```diff
@@ -779,6 +781,9 @@ $(BUILD_DIR)/CACHEDIR.TAG:
 %.bin: %.elf
 	$(Q)$(OBJCOPY) $(OFLAGS) -Obinary $< $@
 
+%.lst: %.elf
+	$(Q)$(OBJDUMP) -DfFhlpSw $< > $@
```

Comment:
Mind using the long  form argument form here? No need to keep this as short as possible and force the reader to look up the arguments.

Reply:
I think this is the long form of all arguments:
```suggestion
	$(Q)$(OBJDUMP) --disassemble-all --file-headers --file-offsets --section-headers --line-numbers --private-headers --source --wide $< > $@
```